### PR TITLE
Auth ui

### PR DESCRIPTION
**Pull request by boyedokup**
## Description

All pull requests should be related to open issues. Indicate the issue(s) bellow and remove this line:
Fixes #(3) #(4)

## How Has This Been Tested?
Compared the design to UI/UX

## Screenshots (if applicable, else remove this line / section)
image
image
image

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/app/service/auth.service.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            this.user = stored_user;
+        } else {
+            return;
+        }
+
+    }
+
+    logout() {
+        this.user = null;
+        localStorage.removeItem('auth_user');
```
Comment (paajake) [inline src/app/service/auth.service.ts]: You should make a post request to the /api/logout end point to also remove the token from the backend, when logging out

**Review by paajake — CHANGES_REQUESTED** [review]

File: src/app/register/register.page.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <ion-row class="ion-justify-content-center">
+            <ion-item>
+                <ion-label position="floating">
+                    Full Name</ion-label>
+                <ion-input type="text" [(ngModel)]="name"></ion-input>
+            </ion-item>
+
+            <ion-item>
+                <ion-label position="floating">
+                    Mobile Number</ion-label>
```
Comment (paajake) [inline src/app/register/register.page.html]: Hide/comment out mobile numbers for now

**Review by paajake — CHANGES_REQUESTED** [review]

File: src/app/register/register.page.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            </ion-item>
+            <ion-item>
+                <ion-label position="floating">
+                    Email</ion-label>
+                <ion-input type="email" [(ngModel)]="email"></ion-input>
+            </ion-item>
+
+            <ion-item>
+                <ion-label position="floating">
+                    Password</ion-label>
```
Comment (paajake) [inline src/app/register/register.page.html]: Add a password_confirmation field

**Comment by paajake** [comment]
Please test the auth and confirm it's working with the live backend on Heroku

**Comment by boyedokup** [comment]
Added the updates to the next pull request

**Review by paajake — CHANGES_REQUESTED** [review]
Wouldn't it be better just to ignore all the IDE and build files? like .idea/

**Review by paajake — APPROVED** [review]

**Pull request merged**